### ISSUE TITLE
H3 Heading

### ISSUE THREAD

**Issue by jbleys** [uweb/uw_wp_theme#50]
Hi,

I was checking out the heading styles, and the h3 is the only one with a 30px margin on top. Is there a specific reason for this?

![2 0 1-headings](https://user-images.githubusercontent.com/2615272/201811223-7191396f-110a-4a5e-8d1c-8096537e9867.JPG)

Thanks,
Jasper Bleijs
Dept of Global Health

**Comment by jbleys** [comment]
Looks like the h3 top margin is removed when the heading is inside a tab, but not in the main content entry.

![2 0 1-headings-1](https://user-images.githubusercontent.com/2615272/202045194-ab6366f9-8ca1-44b3-810e-bc5d77ecfc3b.JPG)

**Comment by lcaple** [comment]
I did some digging on this, but I am not 100% sure if this will be the answer you were looking for:

Short answer: This is carried over from styling in uw-2014, which we did with a lot of legacy css.

Long answer: I don't know exactly what the `h3` has that padding for, but the `h3` style is used in several other elements throughout the classic uw-2014 features for specific visual effects. My best guess is it's there for one of those elements and hasn't caused much of an issue elsewhere so it's never been removed.

my colleagues and I weren't here when the old theme was built and it's quite large, so I don't know the full rationale myself!

We're in the process of going through the theme elements carried over from uw-2014 and things like this will be investigated. For now if you'd like it adjusted, I suggest removing it with inline CSS, customizer code, or within a child theme. But I can't guarantee it won't impact other things in the theme if done globally (like widgets, shortcode output, etc.)

Hope this helps!
